### PR TITLE
Update Bulletin Board Description with new changes in stack

### PR DESCRIPTION
**Pull request by bhattaraib58**
Bulletin Board is not an interactive Kiosk anymore. We have moved on to Web APP. The stack has also been changed to React and Node.

### CODE REVIEW

**Comment by bhattaraib58** [comment]
Hmm, can't add reviewer. 
@kabirbaidhya @sagunji dai please review this PR.

**Comment by sagunji** [comment]
@bhattaraib58 great man. I would also, like to suggest you add the logo of Chautari if there is.

File: README.md [hunk 1]
```diff
@@ -21,9 +21,7 @@ Ever had a situation where you needed to pass a value down to a series of callba
 
 ### [bulletin-board](https://github.com/leapfrogtechnology/bulletin-board)
 
-Leapfrog Bulletin Board is a Web App that helps to build an interactive Kiosk.
-
-Leapfrog Bulletin Board to keep its employees updated with its ongoing activities related to projects, various events, individuals and technologies.
+Leapfrog Bulletin Board acts as a platform for sharing useful information regarding ongoing activities at Company to its employees through different informative segments. Company ongoing activities related to projects, various events, individuals and technologies are shared. Information can be in form of Google slides, videos, images, website and even YouTube Videos.
```
Comment (kabirbaidhya) [inline README.md]:
```suggestion
Leapfrog Bulletin Board acts as a platform for sharing useful information regarding ongoing activities at a company to its employees through different informative segments. Company ongoing activities related to projects, various events, individuals and technologies are shared. Information can be in form of Google slides, videos, images, website and even YouTube videos.
```

**Comment by bhattaraib58** [comment]
> @bhattaraib58 great man. I would also, like to suggest you add the logo of Chautari if there is.

@sagunji  Dai, Chautari is not opensource. I have added the Chautari logo in the README of Chautari.
https://github.com/leapfrogtechnology/chautari

**Comment by sagunji** [comment]
@bhattaraib58 Oops I meant Bulletin Board :D

**Review by kabirbaidhya — APPROVED** [review]

**Pull request merged**